recipes/jenkins-job-builder/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "jenkins-job-builder" %}
+{% set version = "3.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f0c3fc607224c43aa33ec518bfa44e9ae4707063333e9b15091c3181add77585
+
+build:
+  noarch: python
+  number: 0
+  script: "python -m pip install . -vv"
+  
+requirements:
+  host:
+    - python >=3.6
+    - pip
```

According to the [`setup.py`](https://opendev.org/jjb/jenkins-job-builder/src/branch/master/setup.py), you need `pbr` to run build it, so you need to add it here under `host`:
```suggestion
    - pip
    - pbr >=1.8
```